packages/sit-onyx/src/components/OnyxInput/types.ts

```diff
@@ -75,10 +75,16 @@ export type OnyxInputProps = DensityProp &
     loading?: boolean;
     /**
      * Maximum number of characters that are allowed to be entered.
+     * Warning: when the value is (pre)set by code,
```

Comment:
This is true for every validation (also required, custom error etc.) so I don't think we need to document this for every property :)

Reply:
hmm but it also is not detected when you try to send a form.. while the other errors will block sending
![image](https://github.com/SchwarzIT/onyx/assets/151019977/fc544250-e085-413e-b0ca-d953395dc274)

Reply:
Even if you click submit?

Reply:
exactly, that's the issue.. :(
https://github.com/SchwarzIT/onyx/assets/151019977/f898802d-e211-43c1-86b9-c006feaaf702

Reply:
Thats the expected behaviour, if only setting minlength and not required, the form is valid when its empty
![image](https://github.com/SchwarzIT/onyx/assets/67898185/e3782a84-966f-4a86-ac47-0c7ad8cbd48f)

Reply:
in the video you can see that the value is not empty (it's preset by "A") which is what this comment is warning about